### DISCUSSION TITLE
skipAutomaticSimplification & time calculation

### DISCUSSION THREAD

**Discussion by asparamancer** [josdejong/mathjs#2868] [Q&A]
I'm a little unclear on the output of evaluate when on time based strings. I've got:

`
const tempString = 23 hours 16 minutes - 20 hours 00 minutes
`
`
const result = evaluate(tempString)
`
And the result when logged out in the console is is:
`
Unit {fixPrefix: false, skipAutomaticSimplification: false, units: Array(2), dimensions: Array(9), value: 79488000}
dimensions
: 
(9) [0, 0, 2, 0, 0, 0, 0, 0, 0]
fixPrefix
: 
false
skipAutomaticSimplification
: 
false
units
: 
(2) [{…}, {…}]
value
: 
79488000
[[Prototype]]
: 
Object
`
^ Formatter is making it a little hard to read here's a screenshot:
![Screenshot 2022-12-31 at 17 46 41](https://user-images.githubusercontent.com/30260556/210151718-c6b3f9ad-dadd-4fd5-af7b-90cc5a2e845f.png)

I don't see how I can get the value back out in hours and minutes 03:16, or just minutes (196)?

**Comment by josdejong** [comment]
An expression like `23 hours 16 minutes` will be evaluated as an implicit multiplication, like `23 hours * 16 minutes`. You'll have to make it explicit that you want to add hours and minutes, like `23 hours + 16 minutes`.

The returned `Unit` has methods like `.toString()` and `.format()` to get a readable output, you can have a look at the docs.
